helm/api-spec-app/templates/psp-rbac.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: {{ .Values.name }}
+spec:
+    allowPrivilegeEscalation: false
+    fsGroup:
```

Comment:
I feel like theres some indentation wrong here.

Reply:
and in other places (like seLinux for example)

Reply:
Done